crates/services/txpool_v2/src/service.rs

```diff
@@ -382,8 +400,9 @@ where
         let time_txs_submitted = self.pruner.time_txs_submitted.clone();
         let tx_id = transaction.id(&self.chain_id);
         let utxo_validation = self.utxo_validation;
+        let metrics = self.metrics;
 
-        move || {
+        let op_without_metrics = move || {
```

Comment:
A nit (maybe just a naming or merging issue), but there [are metrics](https://github.com/FuelLabs/fuel-core/pull/2321/files#diff-a53ebeb38715cb4803427df1b871c1f7a8a1263f55fd476441b301da5dd27855R434-R437) inside this op without metrics.

Reply:
Yep, `op_without_metrics` is not very useful as a name either way. Moved it back to `op` in 1e3810d69e